requirements.txt

```diff
@@ -0,0 +1 @@
+requests
```

Comment:
Already packaged on PYPI. No need for requirements.txt, anymore. Can you remove this file?